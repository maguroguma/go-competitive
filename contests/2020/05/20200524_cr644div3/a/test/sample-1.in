8
3 2
4 2
1 1
3 1
4 7
1 3
7 4
100 100
