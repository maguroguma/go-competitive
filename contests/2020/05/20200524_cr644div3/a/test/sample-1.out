16
16
4
9
64
9
64
40000
